doc/interfaces/bundle_segmentation_flow.rst

```diff
@@ -0,0 +1,122 @@
+
+Let's say we have an input target tractogram named ``target.trk`` and atlas we
+download named ``whole_brain_MNI.trk``.
+
+Visualizing target and atlas tractograms before registration::
+
+    dipy_horizon "target.trk" "whole_brain_MNI.trk" --random_color
+
+.. figure:: https://github.com/dipy/dipy_data/blob/master/tractograms_initial.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
+
+    Atlas and target tractograms before registration.
+
+------------------------------------
+Streamline-Based Linear Registration
+------------------------------------
+
+For extracting bundles from tractogram we first need our target tractogram to
+be in common space (atlas space). We will register target tractogram to
+model atlas’ space using streamline-based linear registeration (SLR) [3]_.
+
+Following workflows require two positional input arguments; ``Static`` and
+``Moving`` .trk files. ``Static`` would be the ``atlas``  and ``Moving`` would be
+our ``target``  tractogram.
+
+Run the following workflow::
+
+    dipy_slr "whole_brain_MNI.trk" "target.trk" --force
+
+SLR workflow will save transformed tractogram as ``moved.trk``.
+
+Visualizing target and atlas tractograms after registration::
+
+    dipy_horizon "moved.trk" "whole_brain_MNI.trk" --random_color
+
+.. figure:: https://github.com/dipy/dipy_data/blob/master/tractograms_after_registration.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
+
+    Atlas and target tractograms after registration.
+
+-----------
+Recobundles
+-----------
+
+Create an ``out_dir`` folder (eg: rb_output)::
+
+    mkdir rb_output
+
+For Recobundles workflow, we will be using 30 model bundles downloaded earlier.
+Run the following workflow::
+
+    dipy_recobundles "moved.trk" "bundles/*.trk" --force --mix_names --out_dir "rb_output"
+
+This workflow will extract 30 bundles from the tractogram.
+Example of extracted Left Arcuate fasciculus (AF_L) bundle:
+
+.. figure:: https://github.com/dipy/dipy_data/blob/master/AF_L_rb.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
+
+    Extracted Left Arcuate fasciculus (AF_L) from input tractogram
+
+Example of extracted Left Arcuate fasciculus (AF_L) bundle visualized along
+model AF_L bundle used as reference in RecoBundles:
+
+.. figure:: https://github.com/dipy/dipy_data/blob/master/AF_L_rb_with_model.png?raw=true
+    :width: 70 %
+    :alt: alternate text
+    :align: center
+
+    Extracted Left Arcuate fasciculus (AF_L) in Pink and model AF_L bundle in green color.
+
+Output of recobundles will be in native space. To get bundles in subject's
+original space, run following commands::
+
+    mkdir org_output
+
+    dipy_labelsbundles 'target.trk' 'rb_output/*.npy' --mix_names --out_dir "org_output"
+
+
+
+For more information about each command line, you can go to
+`<https://github.com/dipy/dipy/blob/master/dipy/workflows/segment.py>`_
```

Comment:
Maybe link to the website instead of directly to the code? This will probably be intimidating for most users.